compiler/src/dmd/statementsem.d

```diff
@@ -5039,3 +5039,38 @@ bool checkLabel(GotoStatement gs)
     }
     return false;
 }
+
+/*****************************************
+* Perform `static foreach` lowerings that are necessary in order
+* to finally expand the `static foreach` using
+* `dmd.statementsem.makeTupleForeach`.
+*/
+extern(D) void prepare(StaticForeach fs, Scope* sc)
```

Comment:
The extern(D) is implicit here. When `prepare` was a member function this was necessary because the class that contained is extern(C++), but it's not necessary anymore.